samples/newChat/js/app.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+/*
+ * Before start chatting you need to follow this steps:
+ * 1. Initialize QB SDK ( QB.init() );
+ * 2. Create user session (QB.createSession());
+ * 3. Connect to chat in a creqte session callback (QB.chat.connect());
+ * 4. Set listeners;
+ */
+
+function App (config) {
+    this._config = config;
+    this.user = null;
+    this.token = null;
+
+    this.dialogId = null;
+    this.prevDialogId = null;
+
+    // Elements
+    this.page = document.querySelector('#page');
+    this.sidebar = null;
+    this.content = null;
+    this.conversationLinks = null;
+    this.userListConteiner = null;
+    this.init(this._config);
+}
+
+// Before start working with JS SDK you nead to init it.
+
+App.prototype.init = function(config){
+    // Step 1. QB SDK initialization.
+    QB.init(config.credentials.appId, config.credentials.authKey, config.credentials.authSecret, config.appConfig);
+};
+
+App.prototype.setLoginListeners = function() {
+    var self = this,
+        select = document.querySelector('.j-login__select'),
+        loginBtn = document.querySelector('.j-login__button');
+
+    select.addEventListener('change', function(){
+        if(!isNaN(this.value)){
+            loginBtn.removeAttribute('disabled');
+        }
+    });
+
+    loginBtn.addEventListener('click', function(){
+        var userId = +select.value;
+        if (loginBtn.classList.contains('loading')) return false;
+        self.user = _.findWhere(usersList, {id: userId});
+        loginBtn.classList.add('loading');
+
+        self.login();
+
+        loginBtn.innerText = 'loading...'
+    });
+};
+
+App.prototype.login = function(){
+    var self = this,
+        userData = {
+            login: self.user.login,
+            password: self.user.pass
+        };
+
+    // Step 2. Create session with user credentials.
+    QB.createSession(userData, function(err, res) {
+        if (res) {
+            self.token = res.token;
+            // Step 3. Conect to chat.
+            QB.chat.connect({userId: self.user.id, password: self.user.pass}, function(err, roster) {
+                if (err) {
+                    document.querySelector('.j-login__button').innerText = 'Login';
+                    console.log(err);
```

Comment:
Use console.error instead of log
Show user what the problems -  alert('Connect to chat Error'); + description of error